xapian-applications/omega/index_file.h

```diff
@@ -126,6 +126,16 @@ void
 index_add_document(const std::string & urlterm, time_t last_altered,
 		   Xapian::docid did, const Xapian::Document & doc);
 
+void
+index_ctime_terms(Xapian::Document & doc, DirectoryIterator & d,
+		  const time_t & mtime, const time_t & ctime);
+
+/// Update a document without re-extracting text
+bool
+index_update_entry(const std::string & urlterm,
+		   DirectoryIterator & d,
+		   Xapian::docid & did);
```

Comment:
Perhaps the same applies here too?  Less sure about this one, but it's not currently used outside of `index_file.cc`.

Reply:
It is only used on `index_file.cc`. I will add static to the definition and remove it from the header.